api/v1/lib/httpcli/httpsched/httpsched.go

```diff
@@ -134,6 +137,14 @@ func AllowReconnection(v bool) Option {
 	}
 }
 
+func MasterCandidates(cs CandidateSelector) Option {
```

Comment:
nit: maybe rename to `EndpointCandidates` for clarity (since it's actually selecting the next endpoint for the http client)